<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Productos</class>
 <widget class="QDialog" name="Productos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Productos a su nueva Compra</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="table_productos">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_cancelar">
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_detalle">
         <property name="text">
          <string>Ver Detalle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_finalizar">
         <property name="text">
          <string>Finalizar Compra</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>148</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nuevo">
         <property name="text">
          <string>Nuevo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_agregar">
         <property name="text">
          <string>Agregar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_cancelar</tabstop>
  <tabstop>btn_detalle</tabstop>
  <tabstop>btn_finalizar</tabstop>
  <tabstop>btn_nuevo</tabstop>
  <tabstop>btn_agregar</tabstop>
  <tabstop>table_productos</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
